cpuinfo.go

```diff
@@ -464,7 +464,7 @@ func parseCPUInfoRISCV(info []byte) ([]CPUInfo, error) {
 }
 
 func parseCPUInfoDummy(_ []byte) ([]CPUInfo, error) { // nolint:unused,deadcode
-	return nil, errors.New("not implemented")
+	return nil, fmt.Errorf("not implemented")
```

Comment:
This could just stay `errors.New`, but if you want to replace this case, too, I'd do it consistently (see comment below).